docs/source/reference/scipy_special.rst

```diff
@@ -13,6 +13,7 @@ Bessel functions
 
    j0
    j1
+   jn
```

Comment:
Let's keep this unlisted from the docs, as SciPy doesn't list it.

Reply:
Fixed!